### ISSUE TITLE
Update the vulnerable packages

### ISSUE THREAD

**Issue by takaleketan** [jhnns/rewire#194]
Please can you update the vulnerable npm packages to the latest ones

**Comment by sagital** [comment]
I am having the same problem 

`Inefficient Regular Expression Complexity in chalk/ansi-regex`

**Comment by amaciejk** [comment]
Same here, npm ls shows:

```
└─┬ rewire@5.0.0
  └─┬ eslint@6.8.0
    ├─┬ strip-ansi@5.2.0
    │ └── ansi-regex@4.1.0
    └─┬ table@5.4.6
      └─┬ string-width@3.1.0
        └─┬ strip-ansi@5.2.0
          └── ansi-regex@4.1.0
```

This needs an eslint bump to ideally latest 8.3.0:
https://www.npmjs.com/package/eslint

**Comment by langthiennhai** [comment]
I am having the same problem

 1.
  ///////////////////////////////////////////
  Moderate         Inefficient Regular Expression Complexity in chalk/ansi-regex                                              
  Package         ansi-regex                                                    
  Patched in      >=5.0.1                                                       
  Dependency of   rewire                                                        
  Path            rewire > eslint > strip-ansi > ansi-regex         
 ///////////////////////////////////////////

2.
  ///////////////////////////////////////////
  Moderate         Inefficient Regular Expression Complexity in  chalk/ansi-regex                                              
  Package         ansi-regex                                                    
  Patched in      >=5.0.1                                                       
  Dependency of   rewire                                                        
  Path            rewire > eslint > table > string-width > strip-ansi > ansi-regex 
 ///////////////////////////////////////////

**Comment by atishayjainmono** [comment]
+1

**Comment by atishayjain** [comment]
+1

**Comment by oneeyedziggy** [comment]
> latest 8.3.0

looks like at least 8.2.0, but now latest is 8.4.1, neither of which (currently) has no known vulnerabilities

**Comment by jhnns** [comment]
Published as v6.0.0 🚀
